public/app/features/migrate-to-cloud/onprem/MigrationSummary.tsx

```diff
@@ -79,14 +85,20 @@ export function MigrationSummary(props: MigrationSummaryProps) {
           {totalCount}
         </MigrationInfo>
 
-        <MigrationInfo title={t('migrate-to-cloud.summary.errored-resource-count', 'Errors')}>
-          {errorCount}
-        </MigrationInfo>
-
         <MigrationInfo title={t('migrate-to-cloud.summary.successful-resource-count', 'Successfully migrated')}>
           {successCount + warningCount}
         </MigrationInfo>
 
+        <MigrationInfo title={t('migrate-to-cloud.summary.errored-resource-count', 'Errors')}>
+          {errorCount}
+          <Space h={5} layout="inline" />
+          <Checkbox
```

Comment:
What about hiding this checkbox unless there's snapshot results? Showing it while you have a pending snapshot doesn't provide any value

Reply:
Done, will only show checkbox if PROCESSING, PENDING_PROCESSING, or FINISHED